azure-pipelines-dependency-graph-testing.yml

```diff
@@ -360,7 +360,7 @@ stages:
           - script: |
               # These are tests that are specific to Great Expectations Cloud.
               # In order to ensure coverage, we run them during each CI/CD cycle.
-              pytest -m cloud
+              pytest -m cloud --cloud
```

Comment:
Enables Cloud tests but also narrows the collected tests to just those items.

Reply:
Just wanting to clarify what is the difference between `-m cloud` and `--cloud`?

Reply:
`-m` will ONLY collect tests that have that marker. `--cloud` will enable them to actually be run.

```
pytest --cloud                 # Run ALL tests (including Cloud ones)
pytest -m cloud               # Collect Cloud tests (but do not run due to not having --cloud)
pytest --cloud -m cloud  # Collect Cloud tests AND run them
```

If there's a more elegant solution here, please let me know :)

Reply:
works for me for now :) Thanks for teh additional clarity

Reply:
This was very useful for me too!